--- libc/misc/time/time.c.good	2013-10-16 18:31:18.503345098 -0400
+++ libc/misc/time/time.c	2013-10-16 18:31:34.583344849 -0400
@@ -605,11 +605,11 @@
 } ll_tzname_item_t;
 
 /* Structures form a list "UTC" -> "???" -> "tzname1" -> "tzname2"... */
-struct {
+static struct {
 	struct ll_tzname_item *next;
 	char tzname[4];
 } ll_tzname_UNKNOWN = { NULL, "???" };
-const struct {
+static const struct {
 	struct ll_tzname_item *next;
 	char tzname[4];
 } ll_tzname_UTC = { (void*)&ll_tzname_UNKNOWN, "UTC" };
